lib/examples/npdsch_enodeb.c

```diff
@@ -180,7 +186,7 @@ void parse_args(int argc, char** argv)
 
   if (!output_file_name && rf_freq == 0) {
     usage(argv[0]);
-    printf("\nError! Either RF frequency or output filename need to be specified.\n");
+    printf("\nError : either RF frequency or output filename needs to be specified\n");
```

Comment:
nitpick, but we'd prefer `Error: either ...`, so no space behind error

Reply:
No problem Andre - will amend